### PR TITLE
Fix links in cloud printing documentation

### PR DESCRIPTION
**Pull request by nilakhan**

### CODE REVIEW

File: api-reference/beta/api/printdocument-uploaddata.md [hunk 1]
```diff
@@ -1,107 +0,0 @@
----
```
Comment (Lauragra) [inline api-reference/beta/api/printdocument-uploaddata.md:1]: Why are we deleting this topic?

File: api-reference/beta/api/printer-getcapabilities.md [hunk 2]
```diff
@@ -1,142 +0,0 @@
----
-title: "printer: getCapabilities"
```
Comment (Lauragra) [inline api-reference/beta/api/printer-getcapabilities.md:2]: Why are we deleting this one, too?

File: api-reference/beta/api/printjob-get.md [hunk 3]
```diff
@@ -28,10 +28,17 @@ In addition to the following permissions, the user or app's tenant must have an
 
 ## HTTP request
 <!-- { "blockType": "ignored" } -->
+
+To get job using printer:
```
Comment (braedenp-msft) [inline api-reference/beta/api/printjob-get.md:32]: "To get a job from a printer"

File: api-reference/beta/api/printjob-get.md [hunk 4]
```diff
@@ -28,10 +28,17 @@ In addition to the following permissions, the user or app's tenant must have an
...
 ```http
 GET /print/printers/{id}/jobs/{id}
 ```
 
+To get job using printer share:
```
Comment (braedenp-msft) [inline api-reference/beta/api/printjob-get.md:37]: "To get a job from a printer share"

File: api-reference/beta/resources/printdocument.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ Represents a document being printed.
 
 | Method       | Return Type | Description |
 |:-------------|:------------|:------------|
-| [Create upload session](../api/printdocument-get-file.md) | [uploadSession](uploadsession.md) | Create an upload session to iteratively upload ranges of binary file of **printDocument**. |
+| [Create upload session](../api/printdocument-createuploadsession.md) | [uploadSession](uploadsession.md) | Create an upload session to iteratively upload ranges of binary file of **printDocument**. |
```
Comment (braedenp-msft) [inline api-reference/beta/resources/printdocument.md:22]: Recommend: 
> Create an upload session to iteratively upload ranges of a binary file to the **printDocument**.

**Comment by braedenp-msft** [comment]
@nilakhan To save an extra PR, can you also update `print-list-connectors.md` to remove the `$orderby` query parameter restriction? It supports sorting now.

**Review by braedenp-msft — APPROVED** [review]

File: api-reference/beta/api/printer-getcapabilities.md (hunk 2 above)
Comment (nilakhan) [reply api-reference/beta/api/printer-getcapabilities.md:2]: This is also deprecated. 
As per our discussion offline, I will not delete the methods getCapabilities and uploadData in this PR and work on their deprecation process.

File: api-reference/beta/toc.yml [hunk 6]
```diff
@@ -1637,22 +1637,10 @@ items:
             href: api/printer-list-jobs.md
           - name: Create print job
             href: api/printer-post-jobs.md
-          - name: List allowed users
```
Comment (Lauragra) [inline api-reference/beta/toc.yml:1640]: Why are we removing these entries from the TOC? These files exist in the repo.

Comment (nilakhan) [reply api-reference/beta/toc.yml:1640]: I thought I had deleted those files. 
Deleting them now.
These APIs are also being removed from metadata in this week's deployment.
They have been replaced by similar APIs in printerShare.

**Pull request closed without merging**